style.css

```diff
@@ -1,36 +1,860 @@
-@media screen and (min-width: 375px) {
-   
-    #ham {
-    
-     font-size: 35px;
-     margin-left: 20px;
-    }
-    .main-section {
-        position: relative;
-        background-image: url(Img/background.jpg);
-        background-repeat: no-repeat;
-        background-size: 100%;
-        margin-left: -10px;
-        margin-right: -20px;
-
-    }
-    .middle {
-        color: orangered;
-        width: 50%;
-    }
-    .top {
-        color: orangered;
-    }
-    .main-text {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-    }
-    .lower {
-        width: 50%;
-        background-color: rgb(218, 218, 218);
-        font-size: 20px;
-    }
-
-}
\ No newline at end of file
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.fas {
+  color: black;
+}
+
+.removed {
+  display: none;
+}
+
+.d-grid {
+  display: grid;
+  row-gap: 3%;
+}
+
+/* mobile version */
+@media screen and (max-width: 768px) {
+  .mobile-bg {
+    background: url(img/mobile-home-bg.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-size: cover;
+  }
+
+  /* #Program {
+    display: none;
+  } */
+  .featured-speakers2 {
+    display: none;
+  }
+
+  .mobiles-menu {
+    list-style-type: none;
+    overflow: hidden;
+    background: url(img/mobile-home-bg.png);
+    background-repeat: no-repeat;
+    height: 100vh;
+    width: 100%;
+    padding-left: 30px;
+    padding-top: 35px;
+    background-color: white;
+  }
+
+  .mobiles-menu li {
+    padding: 20px;
+  }
+
+  .mobiles-menu li a {
+    font-size: 32px;
+    font-family: "Lato", sans-serif;
+    line-height: 44px;
+    color: #000;
+  }
+
+  #horizontal {
+    margin-right: 50px;
+    border-color: orangered;
+    border-width: 1px;
+  }
+
+  .main-nav {
+    display: none;
+  }
+
+  #head-h3 {
+    display: none;
+  }
+
+  .navigation {
+    display: none;
+  }
+
+  #ham {
+    font-size: 35px;
+    margin-left: 3%;
+  }
+
+  #hide {
+    margin-left: 3%;
+  }
+
+  .main-section {
+    height: 90vh;
+  }
+
+  .middle {
+    font-family: "Lato", sans-serif;
+    font-size: 45px;
+    font-weight: bolder;
+    color: orangered;
+  }
```

Comment:
- [ ] Please try as much as possible to ensure that there is an image in the title text to ensure that your design matches the original design. In addition, this will improve the UI of your application 👍 

Your design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/52aa9cc7-deed-4698-aaf4-f745615eb315)

Original design 😎 
![image](https://github.com/Zel-hub7/Capstone-project1/assets/99394374/94a28a9a-7c2e-4b6d-984b-c195522bf6b0)